src/App.js

```diff
@@ -37,7 +37,7 @@ const tweetsArray = [
 function App() {
   return (
     <div className="App">
-      <Tweet />
+      <Tweet tweet={tweetsArray} />
```

Comment:
Aca podemos hacer el map en vez de pasar todo el arreglo a tweets y reutilizamos el componente de tweet lo mas que se pueda, ya que de la manera en que tu lo resolviste tendria que recibir siempre un array 👀 
```suggestion
<div className="App">
      <Tweet />
      {tweetsArray.map((tweet, i) => (
        <Tweet key={i} tweet={tweet} />
      ))}
    </div>```